src/data/skill-desc.csv

```diff
@@ -51,9 +51,9 @@
 "レース終盤に前方が詰まると\nお告げの力で道を切り開く","<size=22>When the front is packed during the last leg of the race, the oracle's power will open a path</size>"
 "レース終盤で3番手の時に負けそうにな\nると秘めた闘志に火が付き速度が上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度がすごく上がる","<size=22>After proceeding through the race calmly, your speed drastically increases during the last 200m</size>"
-"中盤で2回スキルを使うと\nよりリラックスして持久力が回復する",""
+"中盤で2回スキルを使うと\nよりリラックスして持久力が回復する","<size=22>Restore endurance if you activate two skills in the middle part of the race</size>"
 "最終直線で前の方で追いすがっていると\n不屈の闘志で速度が上がる","<size=22>Increase speed in the last straight when chasing someone close to the lead</size>"
-"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる",""
+"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In the second half of the race, increase speed when you recover endurance with your skills while being in front</size>"
```

Comment:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nレッツラゴー！と速度を上げる","<size=22>In second half of the race, increase speed when you recover endurance with your skills while being in front</size>"
```

Reply:
This is so long… I think only this can be removed without making it unclear

Reply:
"In race's second half" could make it shorter by changing word order around a bit?
Also "increase speed when recovering endurance with skills", going to submit a suggested change